site_tmpl/song_requests.html

```diff
@@ -0,0 +1,107 @@
+{% extends 'base_admin.html' %}
```

Comment:
If at all possible, it would be nice to add a `<meta http-equiv="refresh" content="30"/>` or something, to keep the list of song requests current.